test/adapter/json_api/include_data_if_sideloaded_test.rb

```diff
@@ -146,20 +146,25 @@ def test_block_relationship
           end
 
           def test_node_not_included_when_no_link
-            expected = nil
-            assert_relationship(:unlinked_tags, expected)
+            expected = { meta: {} }
+            assert_relationship(:unlinked_tags, expected, key_transform: :unaltered)
```

Comment:
this was returning `nil` before since the key was `unaltered-tags`. I'm not sure if `meta: {}` is the desired behavior, but it's what the current behavior is.